src/main/java/org/broadinstitute/consent/http/resources/DatasetResource.java

```diff
@@ -370,6 +371,23 @@ public Response getDataset(@PathParam("datasetId") Integer datasetId) {
     }
   }
 
+  @GET
+  @Path("/study/{studyId}")
+  @Consumes("application/json")
+  @Produces("application/json")
```

Comment:
Nit: We don't do this consistently and so it's a general problem in the repo, but we should strive to use the constants instead of raw strings as that can easily be typo-ed
```suggestion
  @Produces(MediaType.APPLICATION_JSON)
```